apps/docs/content/guides/database/extensions/pg-safeupdate.mdx

```diff
@@ -10,19 +10,17 @@ The `pg-safeupdate` extension is a useful tool for protecting data integrity and
 
 ## Enable the extension
 
-`pg-safeupdate` can be enabled on a role basis (where `anon` is the role we want to modify in this example):
-
-{/* prettier-ignore */}
-```sql
-GRANT SET ON PARAMETER session_preload_libraries TO anon;
-ALTER ROLE anon SET session_preload_libraries = 'safeupdate';
-```
+<Admonition type="warning" className="mt-3">
+  Your project's Postgres version needs to be 15.1.1.73+ to use this extension. You can check and
```

Comment:
Putting blank lines between the tag and the content will stop Prettier from breaking the lines (which also breaks them in the final HTML):

<img width="872" alt="image" src="https://github.com/supabase/supabase/assets/26616127/81b19117-1e66-482a-aeb2-0093ed046110">